extension/js/background_page/bg-handlers.ts

```diff
@@ -45,6 +45,15 @@ export class BgHandlers {
     }
   };
 
+  public static createAlarmWithDelay = async ({ alarmName, ms }: Bm.CreateAlarmWithDelay) => {
+    // Create the alarm with delay
+    void chrome.alarms.create(alarmName, { when: Date.now() + ms });
+  };
+
+  public static clearAlarm = async ({ alarmName }: Bm.ClearAlarm) => {
+    void chrome.alarms.clear(alarmName);
+  };
+
```

Comment:
this should be in https://github.com/FlowCrypt/flowcrypt-browser/blob/05ca61543f39dca62997b3554adbba9e1559a182/extension/js/service_worker/bg-handlers.ts